lua/modules/particles.lua

```diff
@@ -60,20 +65,33 @@ particles.newEmitter = function(_, config)
 			p.Physics = particles.fromConfigOrDefault(conf.physics, true)
 			p.Velocity = particles.fromConfigOrDefault(conf.velocity, Number3(0, 0, 0))
 			p.Position = self.Position + particles.fromConfigOrDefault(conf.position, Number3(0, 0, 0))
+			p.Rotation = particles.fromConfigOrDefault(conf.rotation, Rotation(0, 0, 0))
 			p.Acceleration = particles.fromConfigOrDefault(conf.acceleration, Number3(0, 0, 0))
-			p.Scale = particles.fromConfigOrDefault(conf.scale, 1.0)
+			p.startScale = particles.fromConfigOrDefault(conf.start_scale, particles.fromConfigOrDefault(conf.scale, 1.0))
+			p.endScale = particles.fromConfigOrDefault(conf.end_scale, particles.fromConfigOrDefault(conf.start_scale, particles.fromConfigOrDefault(conf.scale, 1.0)))
 			p.Mass = particles.fromConfigOrDefault(conf.mass, 1.0)
 			p.Palette[1].Color = particles.fromConfigOrDefault(conf.color, Color.White)
 			p.CollisionGroups = particles.fromConfigOrDefault(conf.collisionGroups, nil)
 			p.CollidesWithGroups = particles.fromConfigOrDefault(conf.collidesWithGroups, nil)
+			p.startLife = particles.fromConfigOrDefault(conf.life, 10.0)
```

Comment:
Considering my comment about Tick, it's not necessary to add this `startLife` field.